Resources/Private/Templates/Tea/Index.html

```diff
@@ -7,6 +7,17 @@ <h2>
             <f:translate key="plugin.tea.heading"/>
         </h2>
 
+        <f:if condition="{teas}">
+            <f:then>
+                {f:render(section: 'Teas', arguments: {teas: teas})}
```

Comment:
Please let's use the XML syntax when using Fluid view helpers as much as possible. This allows auto-indenting and code completion.